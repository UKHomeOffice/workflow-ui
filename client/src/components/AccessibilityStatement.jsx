import React from 'react';

const AccessibilityStatement = () => (
  <div className="govuk-width-container">
    <main className="govuk-main-wrapper">
      <div className="govuk-grid-row">
        <div className="govuk-grid-column-two-thirds">
          <h1 className="govuk-heading-xl">Accessibility statement for internal Home Office systems</h1>
          <p className="govuk-body">We want everyone to be able to do what they need to do with our internal systems, regardless of their access needs, due to a disability or condition.</p>
          <p className="govuk-body">This page describes:</p>
          <ul className="govuk-list govuk-list--bullet">
            <li>how accessible our internal systems are, including tools and intranets and extranets</li>
            <li>what do if you have a problem</li>
            <li>what we are doing to meet the regulations</li>
          </ul>
          <p className="govuk-body">This accessibility statement applies to Home Office internal systems.</p>

          <h2 className="govuk-heading-l">Using our internal systems</h2>
          <p className="govuk-body">We want as many people as possible to be able to use our internal systems. For example, on many of our systems you should be able to:</p>
          <ul className="govuk-list govuk-list--bullet">
            <li>change colours, contrast levels, zoom and fonts</li>
            <li>complete tasks without a mouse by using input controls such as speech or a keyboard</li>
            <li>listen using a screen reader</li>
          </ul>
          <p className="govuk-body">We’ve also tried to make the content simple to understand.</p>
          <p className="govuk-body">
            There are many ways of making your device easier to use if you have access needs. Find out more on the
            <a href="https://mcmw.abilitynet.org.uk/" target="_blank" rel="noopener noreferrer">My Computer My Way site by Abilitynet</a>
            .
          </p>

          <h2 className="govuk-heading-l">How accessible our internal systems are</h2>
          <p className="govuk-body">We aim to meet international accessibility guidelines, however this may not always be possible, or we may have missed a problem.</p>

          <h2 className="govuk-heading-l">What to do if you have difficulty using these systems</h2>
          <p className="govuk-body">If you have difficulty using one of our internal systems, first use the contact details provided on the internal system to ask for help. If you cannot contact the internal system’s team directly:</p>
          <ul className="govuk-list govuk-list--bullet">
            <li>
              if you are on POISE contact
              <a href="https://horizon.fcos.gsi.gov.uk/section/it-services" target="_blank" rel="noopener noreferrer">IT Now</a>
            </li>
            <li>
              if you are in Her Majesty’s Passport Office, contact
              <a href="https://horizon.fcos.gsi.gov.uk/section/it-services/her-majestys-passport-office-admin-it-services" target="_blank" rel="noopener noreferrer">Admin IT services</a>
            </li>
          </ul>
          <p className="govuk-body">As part of providing this service in another way, we may need to send you messages or documents. Tell us how you want us to send messages or documents to you. Tell us if you need them in a different format, for example large print, audio recording or braille.</p>
          <p className="govuk-body">We’re always looking to improve the accessibility of our internal systems. If you find any problems or think we’re not meeting accessibility requirements, use the contact details above to tell us.</p>
          <p className="govuk-body">
            Read
            <a href="https://www.w3.org/WAI/teach-advocate/contact-inaccessible-websites/" target="_blank" rel="noopener noreferrer">tips on contacting organisations about inaccessible sites</a>
            .
          </p>

          <h2 className="govuk-heading-l">If you are unhappy with how we respond to your report</h2>
          <p className="govuk-body">
            In England, Wales and Scotland, the Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, contact the
            <a href="https://www.equalityadvisoryservice.com/" target="_blank" rel="noopener noreferrer">Equality Advisory and Support Service (EASS)</a>
            .
          </p>
          <p className="govuk-body">
            In Northern Ireland, the Equality Commission for Northern Ireland is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, contact the
            <a href="https://www.equalityni.org/help" target="_blank" rel="noopener noreferrer">Equality Commission for Northern Ireland</a>
            .
          </p>

          <h2 className="govuk-heading-l">Technical information about these systems accessibility</h2>
          <p className="govuk-body">Home Office is committed to making our internal systems accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
          <p className="govuk-body">
            Some of our internal systems are either partially compliant or are not compliant with the
            <a href="https://www.w3.org/TR/WCAG21/" target="_blank" rel="noopener noreferrer">Web Content Accessibility Guidelines version 2.1 AA standard</a>
            , due to the non-compliances and exceptions listed below.
          </p>
          <p className="govuk-body">
            Some of our internal systems are not compliant with the
            <a href="https://www.w3.org/TR/WCAG21/" target="_blank" rel="noopener noreferrer">Web Content Accessibility Guidelines version 2.1 AA standard</a>
            . The non-compliances and exceptions are listed below.
          </p>

          <h2 className="govuk-heading-l">Non accessible content</h2>
          <p className="govuk-body">The content listed below is non-accessible for the following reasons.</p>

          <h2 className="govuk-heading-l">Non-compliance with the accessibility regulations</h2>
          <p className="govuk-body">Some people may find parts of our internal systems difficult to use because:</p>
          <ul className="govuk-list govuk-list--bullet">
            <li>the design cannot be overridden by personal settings for colours and fonts</li>
            <li>the design does not adapt effectively to changes in the window size or level of zoom</li>
            <li>there are parts that are difficult or impossible to understand or use with a screen reader</li>
            <li>there are parts that are difficult or impossible to reach or use without a mouse or by touching the screen</li>
            <li>the text is not written in plain English</li>
            <li>the design is confusing or inconsistent</li>
            <li>audio/video elements may not have captions, transcripts, signing or audio description</li>
          </ul>
          <p className="govuk-body">These issues mean that our internal systems may not meet some WCAG 2.1 success criteria.</p>
          <p className="govuk-body">We are working on identifying and resolving as many of these issues as soon as possible. As we do this we will publish accessibility statements for separate systems.</p>

          <h2 className="govuk-heading-l">Disproportionate burden</h2>
          <p className="govuk-body">We have not yet identified internal systems where it would be a disproportionate burden on the Home Office to resolve the non-compliance. If this is the case with some of our internal systems, we will publish full details explaining our decision.</p>

          <h2 className="govuk-heading-l">Content that’s not within the scope of the accessibility regulations</h2>
          <p className="govuk-body">
            The accessibility regulations
            <a href="http://www.legislation.gov.uk/uksi/2018/952/regulation/4/made" target="_blank" rel="noopener noreferrer">do not require internal systems published before 23 September 2019 to be compliant until they undergo a substantial revision</a>
            .
          </p>
          <p className="govuk-body">Some of our old PDF and Word documents may not be accessible in various ways.</p>
          <p className="govuk-body">
            For example, some may not be properly tagged, or do not use headings. This means that they may be hard to navigate using assistive technology such as screen readers. This does not meet
            <a href="https://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms" target="_blank" rel="noopener noreferrer">Web Content Accessibility Guidelines 2.1 success criterion 1.3.1 (Info and Relationships) 2.4.6 (Headings and Labels)</a>
            .
          </p>
          <p className="govuk-body">
            The accessibility regulations
            <a href="http://www.legislation.gov.uk/uksi/2018/952/regulation/4/made" target="_blank" rel="noopener noreferrer">do not require us to fix PDFs or other documents published before 23 September 2018</a>
            {' '}
            if they’re not essential to providing our services.
          </p>
          <p className="govuk-body">The accessibility regulations do not apply to:</p>
          <ul className="govuk-list govuk-list--bullet">
            <li>pre-recorded time-based media such as video and audio</li>
            <li>online maps and mapping services, as long as essential information is provided in an accessible digital manner for maps intended for navigational use</li>
            <li>third-party content that is neither funded nor developed by, nor under the control of Home Office</li>
            <li>reproductions of items in heritage collections that cannot be made fully accessible</li>
            <li>archived content</li>
          </ul>
          <p className="govuk-body">We do not plan to fix this content. However, requests can be made to make specific content accessible for those who need it. For more information use the contact details above.</p>

          <h2 className="govuk-heading-l">How we tested</h2>
          <p className="govuk-body">Some of our internal systems have had more testing than others. Testing has been conducted by Home Office staff and by contracted agencies. Where they have been checked for compliance, it has been against WCAG 2.0 AA and more recently WCAG 2.1 AA. We also increasingly test for compatibility and usability with assistive technologies such as screen readers.</p>

          <h2 className="govuk-heading-l">What we’re doing to improve accessibility</h2>
          <p className="govuk-body">We are reviewing our approach and working towards meeting the regulations by:</p>
          <ul className="govuk-list govuk-list--bullet">
            <li>developing our accessibility capability across the department</li>
            <li>building teams to test for accessibility issues and recommend fixes to suppliers and delivery teams</li>
            <li>continuing to train our staff to create accessible internal systems</li>
            <li>increasing the use of automated and manual accessibility testing in our development process</li>
            <li>continuing to conduct user research with people who have access needs</li>
            <li>putting in place alternative arrangements for those who need them and where feasible making additional adjustments if these are not enough</li>
            <li>putting in place stronger measures to make accessibility a key part of our procurement processes and contracts</li>
          </ul>
        </div>
      </div>
    </main>
  </div>
);

export default AccessibilityStatement;
