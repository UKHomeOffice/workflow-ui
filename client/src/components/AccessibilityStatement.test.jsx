import React from 'react';
import { shallow } from 'enzyme';
import AccessibilityStatement from './AccessibilityStatement';

describe('Accessibility Statement page', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<AccessibilityStatement />);
    expect(wrapper.exists()).toBe(true);
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<AccessibilityStatement />);
    expect(wrapper).toMatchSnapshot();
  });
});
