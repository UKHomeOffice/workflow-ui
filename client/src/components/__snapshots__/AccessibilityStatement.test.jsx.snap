// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accessibility Statement page matches snapshot 1`] = `ShallowWrapper {}`;
